### DISCUSSION TITLE
Do you like JoomCCK logo ?

### DISCUSSION THREAD

**Discussion by andiamine** [JoomCoder-com/JoomCCK#90] [Polls]
<p align="center">
  <img src="https://github.com/JoomCoder-com/JoomCCK/assets/6179568/c1f4e96d-41d2-4a14-8471-57a4462cc731" alt="JoomCCK is a Content Construction Kit for Joomla CMS"/>
</p>

**Comment by LordHelmchen666** [comment]
Is it allowed to use for promotion?

**Reply by andiamine** [reply]
Yes allowed :)
